components/announcement_bar/announcement_bar.jsx

```diff
@@ -119,3 +124,20 @@ export default class AnnouncementBar extends React.PureComponent {
         );
     }
 }
+
+function mapStateToProps(state) {
```

Comment:
Usually we create separate connector files for this stuff. I assume this wasn't done because there's already an `index.js` in this directory. I think it makes sense to separate these components out if we're going to be connecting to redux.